vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -1598,6 +1590,7 @@ private boolean refreshNotifiedMessagesList() {
      */
     public void displayIncomingCallNotification(MXSession session, Room room, Event event, String callId, BingRule bingRule) {
         Log.d(LOG_TAG, "displayIncomingCallNotification : " + callId + " in " + room.getRoomId());
+        final NotificationManagerCompat nm = NotificationManagerCompat.from(EventStreamService.this);
```

Comment:
the call notification must be in foreground to keep the call alive.
FLAG_NO_CLEAR is not enough.